src/actions/guides/http_and_routing/routing_and_params.cr

```diff
@@ -103,7 +103,7 @@ class Guides::HttpAndRouting::RoutingAndParams < GuideAction
     ```
 
     Here, the string from the request path will be returned by the `some_user_id` method. So in this example if 
-    `/users/123-foo` is requested `some_user_id` would return a text response of `Requested user id: 123-foo`.
+    `/users/123-foo` is requested `some_user_id` would return `123-foo`.
```

Comment:
Can you also add that the action would return a text response of `Requested user id: 123-foo`? I'm thinking something like 

> `some_user_id` would return `123-foo`, and the action would return a text response of `Requested user id: 123-foo`

Reply:
Good idea. I think that would help clarify things a bit

Reply:
Updated.

I also wonder if

> would return `123-foo`

Should be

> `some_user_id` would return `"123-foo"`